google-cloud-pubsub/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,350 @@
+# Ruby Pub/Sub V3 Migration Guide
+
+This page summarizes the changes needed to migrate from `google-cloud-pubsub`
+`v2.x` to `google-cloud-pubsub` `v3.x`.
+In line with [Google's Breaking Change Policy](https://opensource.google/documentation/policies/library-breaking-change),
+we plan to support the existing `v2.x` library for 12 months from the `v3.x` release,
+including bug and security patches, but it will not receive new features.
+
+Note that this is a major version bump that includes breaking changes for the
+Ruby library specifically, but the Pub/Sub API itself remains the same.
+
+## Overview
+
+This major version update involves several significant changes to the Ruby Pub/Sub client library:
+
+1. Removal of handwritten resource management APIs:
```

Comment:
One more nit. Consider boldfacing each of these "titles" (e.g. surround each with double-asterisks for markdown). This will make the list easier to skim.